### DISCUSSION TITLE
How to mark recordings like it was possible before 0.14

### DISCUSSION THREAD

**Discussion by ThomasCr** [blakeblackshear/frigate#14986] [Ask A Question] [answered]
Hi, I am missing the marking of recordings with the star, like it was possible before version 0.14

Is it still possible?

**Comment by NickM-27** [comment]
You use exports for this feature

**Comment by ThomasCr** [comment, marked as answer]
sorry, dupplicate:

https://github.com/blakeblackshear/frigate/discussions/13407
